### PR TITLE
Lung terms (3/3): add new terms associated with genes

### LINKED ISSUE #5261: [NTR] New lung terms - from Mattias and Christina
Peter and I met with pulmonary experts Mattias and Christina. We'll add new lung terms from this [spreadsheet](https://docs.google.com/spreadsheets/d/1Sz47Dk5TZvUh4xMEd6GqfHNqQcmI6BgW/edit#gid=361015257).

Related to #3026 

cc @pnrobinson

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
Note: one term was added as a synonym for an existing term:
Immunodeficiency, autosomal recessive due to ZNFX1 deficiency, I believe is equivalent to OMIM:619644

close #5261

### CODE REVIEW

**Comment by nicolevasilevsky** [comment]
@pnrobinson please let me know if you have any comments about these new terms. I'll go ahead and merge them tomorrow if I don't hear any objections. We can always go back and change them later too, if needed.

**Comment by nicolevasilevsky** [comment]
Update - I talked to Peter, we'll review these with Ada before merging

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -457470,6 +457471,148 @@ def: "An inherited metabolic disease that has its basis in the disruption of the
...
 property_value: IAO:0000233 https://github.com/monarch-initiative/mondo/issues/4985 xsd:anyURI
 
+[Term]
+id: MONDO:0800133
+name: autoinflammatory syndrome due to CEBPE gain-of-function
+def: "An autoinflammatory disease with defective neutrophil function caused by a homozygous Arg219His mutation in the transcription factor C/EBPε." [PMID:31201888]
+is_a: MONDO:0019751 {source="https://orcid.org/0000-0002-7371-8158", source="https://orcid.org/0000-0003-0113-912X"} ! autoinflammatory syndrome
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/1836 {source="https://orcid.org/0000-0002-7371-8158", source="https://orcid.org/0000-0003-0113-912X"} ! CEBPE
+relationship: has_modifier HP:0000007 {source="https://orcid.org/0000-0002-7371-8158", source="https://orcid.org/0000-0003-0113-912X"} ! Autosomal recessive inheritance
+property_value: IAO:0000233 seeAlso https://github.com/monarch-initiative/mondo/issues/5261 xsd:anyURI
```
Comment (matentzn) [inline src/ontology/mondo-edit.obo:457481]:
```suggestion
property_value: IAO:0000233 https://github.com/monarch-initiative/mondo/issues/5261 xsd:anyURI
```

Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:457481]: fixed

**Comment by nicolevasilevsky** [comment]
will redo

**Pull request closed without merging**